src/predict/api/helpers/utils.py

```diff
@@ -181,17 +182,13 @@ def string_to_hourly_datetime(my_list):
     return my_list
 
 
-def get_gp_predictions(airqloud):
+def get_all_gp_predictions():
     '''
-    returns pm 2.5 predictions for a particular airqloud
+    returns pm 2.5 predictions for all airqloud
     '''
-    # try:
-    #     client = MongoClient(MONGO_URI)
-    # except pymongo.errors.ConnectionFailure as e:
-    #     return {'message':'unable to connect to database', 'success':False}, 400
-
-    # db = client[DB_NAME]
-    query = {'airqloud':airqloud}
+    
+    today = datetime.today()
```

Comment:
we shouldn't be returning predictions for the last two days. It should be on predictions from the last run of prediction job.

Reply:
Updates have been made to return predictions for the last hour. But the prediction job is not up yet.